### PR TITLE
use hardhat to verify contracts on etherscan

### LINKED ISSUE #89: Verify contract bytecode on etherscan

### PR DESCRIPTION
**Pull request by wildmolasses**
Resolves #89 

The following contracts are now verified:
- https://etherscan.io/address/0x2a108b1810890482fdb3be188616c56dd0170a8e EndaomentAdmin
- https://etherscan.io/address/0x59bf396e21d8e941415ae80ab7347bdf82d22d6e FundFactory
- https://etherscan.io/address/0xf2f5bb5d80413dd1feea6254786465109adf4b9e OrgFactory
- https://etherscan.io/address/0xbfcc973a3662c91cd2c28f575ff201927479b1d8 Fund
- https://etherscan.io/address/0x82ede3557f4044057ecca56dc6e5908053fc92d2 Org

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require("@nomiclabs/hardhat-etherscan");
+require('dotenv').config();
+
+module.exports = {
+  solidity: {
+    version: "0.6.12",
+    settings: {
+    optimizer: {
+        "enabled": false,
```
Comment (rheeger) [inline hardhat.config.js:9]:
```suggestion
        "enabled": true,
```
should we say here that we are optimized seeing as we deployed optimized contracts?

**Review by rheeger — CHANGES_REQUESTED** [review]
see note

File: hardhat.config.js (hunk 1 above)
Comment (wildmolasses) [reply hardhat.config.js:9]: @rheeger, as the contracts successfully verified with `"enabled": false`, I suspect that the deployed bytecode was compiled with the optimizer disabled. Do you happen to have the artifacts from mainnet deploy on your machine? If so, you can check the artifact json and towards the bottom there's a place where the compiler settings live:

image

Comment (rheeger) [reply hardhat.config.js:9]: fair enough...i thought i had but maybe not. don't have the artifacts handy. This works for now.

**Pull request merged**